### DISCUSSION TITLE
I am having issues, with using equations.

### DISCUSSION THREAD

**Discussion by i-Sage** [ra3xdh/qucs_s#1080] [Q&A] [answered]
I am current learning how to use equations. I am following the tutorial: [https://qucs-s-help.readthedocs.io/en/latest/DModel.html](url). I type the equation word for word, but I am getting these errors:

![circuit](https://github.com/user-attachments/assets/62cb9398-ecd8-47b5-a243-dcd3d001d1a6)

![error](https://github.com/user-attachments/assets/49a0c5d2-481b-4b63-bdac-3566aca1eec5)

Can someone please point me in the right direction

**Comment by ra3xdh** [comment, marked as answer]
You are using Qucsator syntax with Nutmeg equations. This will not work. It accepts only SPICE syntax. The `-i(v1)` will represent the current flowing through the source V1. 

Don't use the documentation from the readthedocs as a reliable source. This manual is 100% outdated. The documentation rewrite is in progress #554. See also the list of the actual documentation for Qucs-S here: #788

**Reply by i-Sage** [reply]
Thank you
